pom.xml

```diff
@@ -732,7 +732,7 @@
     <dependency>
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-configuration2</artifactId>
-      <version>2.7</version>
+      <version>2.8.0</version>
```

Comment:
there should also be another commons-configuration2 in non-mapr profile, is that right?

Reply:
I searched for it in the pom, but it is not present. So I thought I would leave it (thinking about future mapr removal)